ui/app/components/totp-edit.js

```diff
@@ -57,29 +64,45 @@ export default class TotpEdit extends Component {
 
   @action
   reset() {
-    this.model.unloadRecord();
-    this.successCallback(null);
+    this.args.model.unloadRecord();
+    this.transitionToRoute(SHOW_ROUTE, this.args.model.name);
```

Comment:
This _shouldn't_ be necessary anymore with some internal ember data improvements in recent upgrades.. Mostly wondering if this may cause weirdness sometimes because we're unloading a record and then relying on that record to transition.

one way to make this a little more resilient (unloadRecord bugs can be tricky, so we can leave that bit for now) is saving the name as a const before we unload

```suggestion
    const { name } = this.args.model
    this.args.model.unloadRecord();
    this.transitionToRoute(SHOW_ROUTE, name);
```